src/ray/raylet/raylet.cc

```diff
@@ -94,6 +94,9 @@ Raylet::Raylet(instrumented_io_context &main_service,
                                                     resource_map.end());
   self_node_info_.set_start_time_ms(current_sys_time_ms());
   self_node_info_.set_is_head_node(is_head_node);
+  // TODO(rickyx): We could also probably add this to a KV map when we support node
```

Comment:
I don't think I understand the comment. Can you give me a bit more details?

Reply:
it's outdated i think.